go.mod

```diff
@@ -8,9 +8,10 @@ require (
 	github.com/go-openapi/spec v0.19.4
 	github.com/google/go-cmp v0.3.1
 	github.com/google/go-github/v29 v29.0.3
-	github.com/kabanero-io/manifestival v0.0.0-20191230200607-194733cd27d9
 	github.com/knative/pkg v0.0.0-20190817231834-12ee58e32cc8 // indirect
 	github.com/knative/serving-operator v0.0.0-20190702004031-e30377b852ff
+	github.com/manifestival/controller-runtime-client v0.1.1-0.20200218204725-1af9550ddf8f
+	github.com/manifestival/manifestival v0.1.1-0.20200219193505-fabb889b98f5
```

Comment:
I guess I need to bite the bullet and figure out go modules. :smile: I'm confused by the release version since I don't recall releasing that version. I'm currently struggling with versions trying to port one of our operator-sdk-generated apps, so I'm guessing you're still using a pretty old (or much newer?) version.

Reply:
Well, let me start by saying that we are still pretty clueless about modules.  We were more or less forced to switch because operator-sdk switched (or at least that's how we understood it to be).  We're on operator-sdk 0.15.1 now, which is pretty current.

In this specific case, we tried to let `go` figure out what version to use, and it picked the latest release in `manifestival/manifestival` which was v0.1.0.  However that didn't have the changes you made to allow multiple sources.  So we've learned that you can tell `go.mod` to use the `master` branch, and then `go` will generate some string that somehow identifies what that is.... in this case I'm assuming it picked `v0.1.1` because that would come next in semver, and then the date and commit hash from where master was at that point in time.  Maybe?

Reply:
Ah, good to know! I was planning to cut a 0.2.0 after I get my dep issue solved, but I guess as long as you have a recent master revision (whatever go.mod wants to call it) working for you, it doesn't matter that much.

BTW, I'm trying to be better about communicating which features are included in which release [here](https://github.com/manifestival/manifestival/blob/master/CHANGELOG.md).